spec/annexes/extension_crs_wkt.adoc

```diff
@@ -155,8 +155,8 @@ CREATE TABLE gpkg_spatial_ref_sys (
   srs_id INTEGER NOT NULL PRIMARY KEY,
   organization TEXT NOT NULL,
   organization_coordsys_id INTEGER NOT NULL,
-  definition  TEXT NOT NULL,
+  definition  TEXT NOT NULL DEFAULT 'undefined',
   description TEXT,
-  definition_12_063 TEXT NOT NULL
+  definition_12_063 TEXT NOT NULL DEFAULT 'undefined'
 );
```

Comment:
I'm not sure what this change is doing here. It is part of https://github.com/opengeospatial/geopackage/pull/334 which was already merged.